src/Microsoft.AspNetCore.Server.IntegrationTesting.IIS/IISExpressDeployer.cs

```diff
@@ -25,16 +26,18 @@ public class IISExpressDeployer : ApplicationDeployer
         private const string FailedToInitializeBindingsMessage = "Failed to initialize site bindings";
         private const string UnableToStartIISExpressMessage = "Unable to start iisexpress.";
         private const int MaximumAttempts = 5;
-
+        private const int ShutdownMilliseconds = 60000;
         private static readonly Regex UrlDetectorRegex = new Regex(@"^\s*Successfully registered URL ""(?<url>[^""]+)"" for site.*$");
 
-        private Process _hostProcess;
-
         public IISExpressDeployer(DeploymentParameters deploymentParameters, ILoggerFactory loggerFactory)
             : base(deploymentParameters, loggerFactory)
         {
         }
 
+        public Process HostProcess { get; set; }
```

Comment:
This should be on deployment result

Reply:
Going to be a bit tricky to wire up, but I agree.

Reply:
`_serverManager.ApplicationPools[0].WorkerProcesses[0].ProcessId`